tldraw/tldraw
# Fix watermark link opening twice

In 37e8df7 on 20 Sep, the watermark link was changed to make it work in VS Code. However, that change introduced a regression which is currently causing the tldraw website to open _twice_ when the watermark link is clicked on web.

This commit changes the `<a>` to not actually be a link, but rather to only trigger using the `onClick` event.

### Change type


### Test plan

1. Open web build.
2. Click on the watermark.
3. tldraw.dev should open once, not twice.

### Release notes

- Fix watermark link opening twice
